extras/dummy_generator/generator.py

```diff
@@ -415,3 +437,43 @@
 
         # Bulk-create the weight entries
         WeightEntry.objects.bulk_create(new_entries)
+
+# Nutrition Generator
+if hasattr(args, 'number_nutrition_plans'):
+    print("** Generating {0} nutrition plan(s) per user".format(args.number_nutrition_plans))
+
+    if args.add_to_user:
+        userlist = [User.objects.get(pk=args.add_to_user)]
+    else:
+        userlist = [i for i in User.objects.all()]
+
+    # Load all ingredients to a list
+    ingredientList = [i for i in Ingredient.objects.all()]
+
+    # Total meals per plan
+    total_meals = 4
+    
+    for user in userlist:
+        print('   - generating for {0}'.format(user.username))
+
+        # Add nutrition plan
+        for i in range(0, args.number_nutrition_plans):
+            uid = str(uuid.uuid4()).split('-')
+            start_date = datetime.date.today() - datetime.timedelta(days=random.randint(0, 100))
+            nutrition_plan = NutritionPlan(language=Language.objects.all()[1], description='Dummy nutrition plan - {0}'.format(uid[1]),
+                              creation_date=start_date)
+            nutrition_plan.user = user
+
+            nutrition_plan.save()
+
+            # Add meals to plan
+            order = 1
+            for j in range(0, total_meals):
+               meal = Meal(plan=nutrition_plan, order=order)
+               meal.save()
+               ingredient = ingredientList[random.randint(0, len(ingredientList))]
```

Comment:
You can add another for loop here and add a random number of ingredients to the meal, perhaps something between 1 and 5?